### PR TITLE
Add fixed NFT supply topic

### PR DESCRIPTION
**Pull request by DennisDawson**

### CODE REVIEW

File: content/concepts/tokens/nft-fixed-supply.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+html: nft-fixed-supply.html
+parent: non-fungible-tokens.html
+blurb: Introduction to XRPL NFTs.
+labels:
+  - Non-fungible Tokens, NFTs
+---
+
+# Guaranteeing a Fixed Supply of NFTs
+
+For some projects, you might want to guarantee that no more than a fixed number of NFTs are minted from a given issuing account. Use the following steps to achieve that goal.
```
Comment (oeggert) [inline content/concepts/tokens/nft-fixed-supply.md]:
```suggestion
For some projects, you might want to guarantee that no more than a fixed number of NFTs are minted from an issuing account.
```

File: content/concepts/tokens/nft-fixed-supply.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+To guarantee a fixed number of NFTs:
+
+1. Create and fund a new account, the _Issuer_. This account is the issuer of the tokens within the collection. See [Creating Accounts](accounts.html#creating-accounts).
+1. Use `AccountSet` to assign your operational wallet as an authorized minter for the issuer. See [Authorizing Another Account to Mint Your NFTs](nftoken-authorized-minting.html).
+1. Use your operational account to mint the tokens using NFTokenMint. The operational wallet holds all of the tokens minted for the Issuer. See [Batch Minting](nftoken-batch-minting.html).
```
Comment (oeggert) [inline content/concepts/tokens/nft-fixed-supply.md]:
```suggestion
1. Use your operational account to mint the tokens using `NFTokenMint`. The operational wallet holds all of the tokens minted for the Issuer. See [Batch Minting](nftoken-batch-minting.html).
```

**Review by oeggert — APPROVED** [review]
Minor suggestions. Looks good otherwise.

**Comment by ckniffen** [comment]
Should it be added that by blackholing the issuer there would be no access to the transfer fees?

**Review by tequdev — COMMENTED** [review]
It would be better to make it clear that in blackholing, you will effectively not receive any royalties.
I think this is a problem that can be solved by Hooks.

**Comment by DennisDawson** [comment]
This PR is messed up. I'm going to start a new PR to see if the problems resolve.

**Pull request closed without merging**